Services/GlobalCache/classes/Setup/class.ilGlobalCacheSetupAgent.php

```diff
@@ -34,7 +34,15 @@ public function getArrayToConfigTransformation() : Refinery\Transformation
     {
         return $this->refinery->custom()->transformation(function ($data) {
             $settings = new \ilGlobalCacheSettings();
-            if ($data === null || !$data["components"] || $data["service"] == "none") {
+            if (
+                $data === null ||
+                !$data["components"] ||
+                $data["service"] == "none" ||
+                (
+                    $data["service"] == "memcached" &&
+                    (!isset($data["memcached_nodes"]) || count($data["memcached_nodes"]) == 0)
```

Comment:
`count($data["memcached_nodes"]) == 0` to `count($data["memcached_nodes"]) === 0`